Futures-in-Scala-2.12-part-4.md

```diff
@@ -0,0 +1,31 @@
+#Futures in Scala 2.12 (part 4)
+
+This is the third of several posts describing the evolution of `scala.concurrent.Future` in Scala `2.12.x`.
+For the previous post, [click here](https://github.com/viktorklang/blog/blob/master/Futures-in-Scala-2.12-part-3.md).
```

Comment:
So the only remaining uncertainty I have about this format is whether to link to `master` or to a commit / tag for these—which could easily devolve into a discussion on general philosophy :-)